admiral.sh

```diff
@@ -32,6 +32,7 @@ readonly AWSCLI_VERSION=1.10.63
 readonly API_TIMEOUT=600
 readonly ADMIRAL_PORT=50003
 export LC_ALL=C
+export docker_restart=false
```

Comment:
i dont think we need to do this for local installation. once docker boots up, everything else should just work without changes. Setting docker opts and restarting is unnecessary. unless im missing something here?